packages/botonic-plugin-dynamodb/package.json

```diff
@@ -5,7 +5,7 @@
     "test": "../../node_modules/.bin/jest",
     "lint": "npm run lint_core -- --fix",
     "lint_ci": "npm run lint_core -- -c .eslintrc_slow.js",
-    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.ts*' 'tests/**/*.ts*'",
+    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet '.*.js' 'src/**/*.ts*' 'tests/**/*.ts*'",
```

Comment:
One last observation... I guess that here there is no `*.js` because all the files are `*.ts`, right?
Now I think I see the differences among the several linter files

Reply:
yes, in some places there are babel.config.js or similar. I think that if you put a glob which finds nothing it complains, so I have to tailor it to each project